sql/load_demo_data.sql

```diff
@@ -6286,14 +6493,14 @@ DROP TABLE IF EXISTS `oh_user_settings`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `oh_user_settings` (
-   `USS_ID` int(11) NOT NULL AUTO_INCREMENT, 
-   `USS_US_ID_A` varchar(50) NOT NULL, 
-   `USS_CONFIG_NAME` varchar(50) NOT NULL,  
-   `USS_CONFIG_VALUE` text NOT NULL,
-   PRIMARY KEY (`USS_ID`),
-   FOREIGN KEY (`USS_US_ID_A`) REFERENCES OH_USER(`US_ID_A`) ON DELETE CASCADE,
-   UNIQUE (`USS_US_ID_A`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3 COLLATE=utf8mb3_general_ci;
+  `USS_ID` int(11) NOT NULL AUTO_INCREMENT,
```

Comment:
The version in the multiple step empty database is: 
<pre>
 `USS_ID` int(11) unsigned NOT NULL AUTO_INCREMENT,
</pre>

Reply:
Same as above, aligned step_96